src/i18n/nb_NO.ts

```diff
@@ -0,0 +1,1233 @@
+        },
+    },
+    vehicles: {
+        0: {
+            caption: 'Mannskapsbil',
+            color: '#cc0000',
+            coins: 25,
+            credits: 5_000,
+            minPersonnel: 1,
+            maxPersonnel: 6,
+            wtank: 0,
+        },
+        1: {
+            caption: 'Lett mannskapsbil',
+            color: '#bb0000',
+            coins: 25,
+            credits: 5_000,
+            minPersonnel: 1,
+            maxPersonnel: 3,
+            wtank: 0,
+        },
+        2: {
+            caption: 'Stigebil',
+            color: '#d92626',
+            coins: 30,
+            credits: 10_000,
+            minPersonnel: 1,
+            maxPersonnel: 3,
+            wtank: 0,
+            special: 'Nødvendig etter at du har bygd 3 brannstasjoner.',
+        },
+        3: {
+            caption: 'Innsatslederbil brann',
+            color: '#d02525',
+            coins: 25,
+            credits: 10_000,
+            minPersonnel: 1,
+            maxPersonnel: 3,
+            wtank: 0,
+            special: 'Nødvendig etter at du har bygd 6 brannstasjoner.',
+        },
+        4: {
+            caption: 'Tungredningsbil',
+            color: '#ad1f1f',
+            coins: 25,
+            credits: 12_180,
+            minPersonnel: 1,
+            maxPersonnel: 4,
+            wtank: 0,
+            special: 'Nødvendig etter at du har bygd 4 brannstasjoner.',
+        },
+        5: {
+            caption: 'Ambulanse',
+            color: '#9c1c1c',
+            coins: 25,
+            credits: 5_000,
+            minPersonnel: 1,
+            maxPersonnel: 3,
+            wtank: 0,
+        },
+        6: {
+            caption: 'Røykdykkerbil',
+            color: '#aa0000',
+            coins: 25,
+            credits: 11_860,
+            minPersonnel: 1,
+            maxPersonnel: 3,
+            wtank: 0,
+            special: 'Nødvendig etter at du har bygd 5 brannstasjoner.',
+        },
+        7: {
+            caption: 'Tankbil',
+            color: '#990000',
+            coins: 25,
+            credits: 17_300,
+            minPersonnel: 1,
+            maxPersonnel: 3,
+            wtank: 3_000,
+            special: 'Nødvendig etter at du har bygd 7 brannstasjoner.',
+        },
+        8: {
+            caption: 'Redningsbil',
+            color: '#880000',
+            coins: 25,
+            credits: 12_180,
+            minPersonnel: 1,
+            maxPersonnel: 4,
+            wtank: 0,
+            special: 'Nødvendig etter at du har bygd 4 brannstasjoner.',
+        },
+        9: {
+            caption: 'CBRNe enhet',
+            color: '#770000',
+            coins: 25,
+            credits: 19_200,
+            minPersonnel: 1,
+            maxPersonnel: 6,
+            wtank: 0,
+            special: 'Nødvendig etter at du har bygd 11 brannstasjoner.',
+            schooling: 'Krever spesialistutdanning (CBRNe-enhet)',
```

Comment:
Here we need the name as it appears in the course window example: https://prnt.sc/vdzggx